ui/components/app/assets/asset-list/asset-list-control-bar/index.scss

```diff
@@ -1,6 +1,6 @@
 .asset-list-control-bar {
-  padding-top: 4px;
-  padding-bottom: 4px;
+  padding-top: 24px;
+  padding-bottom: 8px;
```

Comment:
Added padding to match figma

<img width="927" height="322" alt="Screenshot 2025-08-20 at 22 04 49" src="https://github.com/user-attachments/assets/8e96433f-7ebc-4e11-9c2c-c7fd6c5ee4f1" />